app/Resources/views/admin/blog/index.html.twig

```diff
@@ -19,7 +19,7 @@
             <tr>
                 <td>{{ post.title }}</td>
                 <td>{{ post.authorEmail }}</td>
-                <td>{% if post.publishedAt %}{{ post.publishedAt|date('Y-m-d H:i') }}{% endif %}</td>
+                <td>{% if post.publishedAt %}{{ post.publishedAt|localizeddate('short', 'short', app.request.get('_locale')|default('en')) }}{% endif %}</td>
```

Comment:
no need to pass the locale explicitly. The Request object sets the Intl locale properly already